### PR TITLE
[show] vnet advertised-route command

### PR DESCRIPTION
**Pull request by siqbal1986**
implementation and tests for "show vnet advertised-routes ip/ipv6 [string]"
#### What I did
Added a new show command to see the VNET Tunnel routes being advertised by BGP. THe output can be filtered based on IPv4/Ipv6 and community string.
#### How I did it
Iterate over the BGP_PROFILE_TABLE in Application DB and ADVERTISE_NETWORK_TABLE in state DB. Correlate the information and filter based on community string.
#### How to verify it
run **show vnet advertised-routes ** with optional community string.
```
$ show vnet advertised-route
Prefix                    Profile              Community Id
------------------------  -------------------  --------------
160.62.191.1/32           FROM_SDN_SLB_ROUTES  1234:1235
160.63.191.1/32           FROM_SDN_SLB_ROUTES  1234:1235
160.64.191.1/32           FROM_SDN_SLB_ROUTES  1234:1235
fccc:a250:a251::a6:1/128
fddd:a150:a251::a6:1/128  FROM_SDN_SLB_ROUTES  1234:1235

$ show vnet advertised-route 1234:1235
Prefix                    Profile              Community Id
------------------------  -------------------  --------------
160.62.191.1/32           FROM_SDN_SLB_ROUTES  1234:1235
160.63.191.1/32           FROM_SDN_SLB_ROUTES  1234:1235
160.64.191.1/32           FROM_SDN_SLB_ROUTES  1234:1235
fddd:a150:a251::a6:1/128  FROM_SDN_SLB_ROUTES  1234:1235

```

#### Previous command output (if the output of a command-line utility has changed)
None
#### New command output (if the output of a command-line utility has changed)
NA

### CODE REVIEW

**Review by prsunny — APPROVED** [review]

File: show/vnet.py [hunk 1]
```diff
@@ -14,6 +15,74 @@ def vnet():
...
+            r.append(profile)
+            r.append(profiles[profile])
+            table.append(r)
+    click.echo(tabulate(table, header))
+
+
+@advertised_routes.command()
+@click.argument('args', metavar='[community:string]', nargs=1, required=False)
+def ip(args):
+    """Show vnet advertised-routes ip [community string XXXX:XXXX]"""
```
Comment (prsunny) [inline show/vnet.py:75]: i recommend 'show vnet advertised-routes' and use '-6' for v6. 'ip' at the end confuses if we need to provide a parameter.

File: tests/mock_tables/state_db.json [hunk 2]
```diff
@@ -890,5 +890,17 @@
...
+    "ADVERTISE_NETWORK_TABLE|160.63.191.1/32": {
+        "profile": "FROM_SDN_SLB_ROUTES"
+    },
+    "ADVERTISE_NETWORK_TABLE|160.62.191.1/32": {
+        "profile": "FROM_SDN_SLB_ROUTES"
+    },
+    "ADVERTISE_NETWORK_TABLE|160.64.191.1/32": {
+        "profile": "FROM_SDN_SLB_ROUTES"
+    },
+    "ADVERTISE_NETWORK_TABLE|fddd:a150:a251::a6:1/128": {
```
Comment (prsunny) [inline tests/mock_tables/state_db.json:903]: please also add a case for no-profile

**Review by prsunny — CHANGES_REQUESTED** [review]

File: tests/mock_tables/state_db.json (hunk 2 above)
Comment (siqbal1986) [reply tests/mock_tables/state_db.json:903]: sure

**Review by prsunny — APPROVED** [review]

**Review by prsunny — APPROVED** [review]

**Comment by yxieca** [comment]
@siqbal1986 this change cannot be cherry-picked to 202205 branch cleanly. Please raise separate PR.

**Comment by yxieca** [comment]
@siqbal1986 this change cannot be cherry-picked to 202205 branch cleanly. Please raise separate PR.

**Pull request merged**